src/test/java/jenkins/plugins/logstash/LogstashIntegrationTest.java

```diff
@@ -72,7 +73,7 @@ public void dataIsSetWhenEnabledViaJobPropertyOnMaster() throws Exception
         FreeStyleBuild build = f.get();
         assertThat(build.getResult(), equalTo(Result.SUCCESS));
         List<JSONObject> dataLines = memoryDao.getOutput();
-        assertThat(dataLines.size(), is(4));
+        assertThat(dataLines.size(), greaterThan(3));
```

Comment:
jenkins log now has 5 messages at this point, I assume you don't really care how many messages there are